tasks.py

```diff
@@ -605,8 +605,6 @@ def build(_c, tool="all", environment="all", trace=False, debug=False, dry_run=F
                 f"Would have run utils.render_jinja2 on {constants.FUNCTIONS_INPUT_FILE}..."
             )
 
-        # TODO: Figure out how to handle -large, somewhere in this function?
```

Comment:
We decided not to do a -large image right now.  Keep the images small and focused